metadata/index.html

```diff
@@ -482,13 +486,10 @@
 }
             </pre>
             <p>
-             The <a>property value</a> of a <a>natural language property</a> is defined as:
-            </p>
-            <ul>
-              <li>if the metadata value is a string, that string</li>
-              <li>if the metadata value is an array, the strings in that array</li>
-              <li>if the metadata value is an object, the string or strings that are the value of the property of that object whose name is value of the <code>lang</code> <a>inherited property</a> on that description, or <code>und</code> if no <code>lang</code> is defined.</li>
-            </ul>
+             The <a>property value</a> of a <a>natural language property</a> is an object whose properties are language codes and where the values of those properties are an array of strings (see <cite><a href="http://www.w3.org/TR/json-ld/#language-maps">Language Maps</a></cite> in [[!JSON-LD]]).</p>
+             <p class="note">
+               When extracting a <a>property value</a> metadata will have already been <a>merged</a>, a <a>natural language property</a> will already have this form.
```

Comment:
Where is it spelled out what this can look like in an unmerged metadata document? I'm worried about losing this information for authors. Perhaps adding some examples would make it clearer?